app/assets/javascripts/static/ContactUsView.js

```diff
@@ -0,0 +1,244 @@
+define([
+  'jquery',
+  'backbone'
+], function($, Backbone) {
+
+  'use strict';
+
+  const topics = {
+    'report-a-bug-or-error-on-gfw': {
+      name: 'Report a bug or error on GFW',
+      placeholder: 'Explain the bug or error and tell us where on the website you encountered it. What browser (e.g., Chrome version 50.0.2661.94 m) and operating system (e.g., Windows 8.1) do you use?',
+    },
+    'provide-feedback': {
+      name: 'Provide feedback',
+      placeholder: 'Tell us about your experience with GFW! Examples: How can we improve GFW? Why did you visit GFW? How do you use GFW? If and how is the information provided by GFW useful for your work? Are there any additional features and/or data that would be useful?  Was anything confusing or difficult to use?  Etc...',
+    },
+    'media-request': {
+      name: 'Media request',
+      placeholder: 'How can we help you?',
+    },
+    'data-related-inquiry': {
+      name: 'Data-related inquiry or suggestion',
+      placeholder: 'How can we help you?',
+    },
+    'gfw-commodities-inquiry': {
+      name: 'GFW Commodities inquiry',
+      placeholder: 'How can we help you?',
+    },
+    'gfw-fires-inquiry': {
+      name: 'GFW Fires inquiry',
+      placeholder: 'How can we help you?',
+    },
+    'gfw-climate-inquiry': {
+      name: 'GFW Climate inquiry',
+      placeholder: 'How can we help you?',
+    },
+    'gfw-water-inquiry': {
+      name: 'GFW Water inquiry',
+      placeholder: 'How can we help you?',
+    },
+    'general-inquiry': {
+      name: 'General inquiry',
+      placeholder: 'How can we help you?',
+    },
+  }
+
+  const constraints = {
+    'contact-email': {
+      presence: true,
+      email: true
+    },
+    'contact-topic': {
+      presence: true
+    },
+    'contact-message': {
+      presence: true
+    }
+  };
+
+  var ContacUsView = Backbone.View.extend({
+
+    el: '#contact_us',
+
+    events: {
+      'click .js-btn-submit': 'actionSubmit',
+      'change input, textarea, select': 'changeInput',
+    },
+
+    initialize: function() {
+      this.$spinner = this.$el.find('.m-spinner');
+      this.$form = this.$el.find('#contact-form');
+      this.$step = this.$el.find('.step');
+      this.$stepBtn = this.$el.find('.step-btn');
+      this.$contactMessage = this.$el.find('#contact-message');
+    },
+
+    actionSubmit(e) {
+      e && e.preventDefault();
+      (this.validate(e)) ? this.sendForm() : this.updateForm();
+    },
+
+    // Send the data to the API
+    sendForm() {
+      // Production send request
+      // Send request
+      this.$spinner.addClass('-start');
+      var xhr = new XMLHttpRequest();
+
+      xhr.open('POST', window.gfw.config.GFW_API_HOST + '/emails');
+      xhr.setRequestHeader('Content-Type', 'application/json; charset=utf-8');
+
+      xhr.onload = function() {
+        if (xhr.status === 200 || xhr.status === 201) {
+          this.changeStep('success');
```

Comment:
Can we scroll to the div's top position? If you are close to the bottom of the page you might not see the success message.